### DISCUSSION TITLE
Updating the store

### DISCUSSION THREAD

**Discussion by Jakobud** [joshnuss/svelte-local-storage-store#152] [Announcements]
Looking at the example
```
export const preferences = persisted('preferences', {
  theme: 'dark',
  pane: '50%',
  ...
})
```

How could I go about changing the `theme` value? What is the syntax?

```
let pref = get(preferences);
pref.theme = 'light';
preferences.set(preferences); // this doesn't seem right
```

Really just kinda not clear from your documentation examples. You just say `preferences.set(...) // set value` but what is that? What value am I setting? Where do I put in the key for the object I want to change?

**Comment by joshnuss** [comment]
Svelte's writable stores come with an `update()` function.

You can mutate state like this:

```javascript
preferences.update(state => {
   return {...state, theme: 'light' } 
})
```

**Reply by gamberoillecito** [reply]
I think this should be added to the readme, it took me a while to figure it out

**Comment by Jakobud** [comment]
Thanks I didn't see the discussions

**Reply by joshnuss** [reply]
No worries! I set them up just now (forgot to turn them on before)
